src/hints/tests.rs

```diff
@@ -0,0 +1,80 @@
+#[cfg(test)]
+mod tests {
+    use std::sync::Arc;
+
+    use blockifier::block_context::{BlockContext, FeeTokenAddresses, GasPrices};
+    use blockifier::transaction::errors::TransactionExecutionError;
+    use blockifier::transaction::objects::TransactionExecutionInfo;
+    use cairo_vm::serde::deserialize_program::ApTracking;
+    use cairo_vm::types::exec_scope::ExecutionScopes;
+    use num_bigint::BigInt;
+    use rstest::{fixture, rstest};
+    use starknet_api::block::{BlockNumber, BlockTimestamp};
+    use starknet_api::core::{ChainId, ContractAddress, PatriciaKey};
+    use starknet_api::hash::StarkHash;
+    use starknet_api::transaction::Fee;
+    use starknet_api::{contract_address, patricia_key};
+
+    use crate::hints::*;
+
+    #[fixture]
+    fn block_context() -> BlockContext {
```

Comment:
Why do you need this here?

Reply:
ExecutionHelperWrapper::new() takes a block_context. Otherwise it's irrelevant

Reply:
Makes sense. Could you put this fixture in some common test code to avoid duplication then?

Reply:
de2cce6 attempts to fix this: I actually copied it from your #4, so de2cce6 makes both cases share this implementation.

I'm open to other suggestions, though, it still seems a bit sloppy...